app/controllers/application_controller.rb

```diff
@@ -1,13 +1,3 @@
 class ApplicationController < ActionController::Base
   before_action :authenticate_user!
-  # protect_from_forgery with: :exception
-
-  # before_action :update_allowed_parameters, if: :devise_controller?
-
-  # protected
-
-  # def update_allowed_parameters
-  #   devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :photo, :bio, :email, :password) }
-  #   devise_parameter_sanitizer.permit(:account_update) { |u| u.permit(:name, :photo, :bio, :email, :password, :current_password) }
-  # end
 end
```

Comment:
- [ ] As stated by the previous reviewer, kindly add the removed code and uncomment it which specifies the allowed params for the devise.
> * [ ]  this shouldn't be commented especially the update_allowed_parameters which allow you to add input in registrations like name bio etc .... so please try to uncomment it to fix it

Reply:
alright